src/jsx/Converter.js

```diff
@@ -25,7 +25,7 @@ import RecordingParameters from './Converter/RecordingParameters';
 const Converter = (props) => {
   const socketContext = useContext(SocketContext);
   const {state, setState, config} = useContext(AppContext);
-
+  console.log(state);
```

Comment:
To remove

Reply:
@NadaElmasry would you have time to briefly clean this up ?  thanks, this would be great before the end of November. 

cc @laemtl @jeffersoncasimir

Reply:
Sorry I didn't receive any notifications for this, I'll work on it this weekend.